packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-line.tsx

```diff
@@ -94,9 +96,10 @@ function SVGLine(props: {
     style: SVGProps<SVGLineElement>["style"];
     dragging?: boolean;
 }) {
-    const {start, end, style, dragging} = props;
+    const {start, end, style, dragging, ...rest} = props;
```

Comment:
Just to pass `data-test-id`, I could also just add `data-test-id` to props if we think that would be better than using `...rest`

Reply:
Although TypeScript will still properly type-check this, I tend to favour explicit props so that it's easier to read and see what exactly is being passed down. But I'm not adamant about this. :)

Reply:
Changed